website/docs/components/copy/button/partials/code/composition.md

```diff
@@ -0,0 +1,49 @@
+## Composition with form inputs
+
+These representative examples showcase the compositional recommendations in the [guidelines](/components/copy/button#composition-with-other-components) using the [Fieldset](/components/form/primitives?tab=code#formfieldset) primitive.
+
+### With single line inputs
+
+```handlebars
+<Hds::Form::Fieldset @layout="vertical" as |F|>
+  <F.Legend>Cluster name</F.Legend>
+  <F.HelperText>This is some example helper text.</F.HelperText>
+  <F.Control>
+    <div style="display: flex; flex-direction: row; gap: 8px; align-items: flex-start;">
```

Comment:
I would not add inline styles like this as I don't think it's something we would want to model and encourage in examples which consumers are likely to copy and paste.

You can instead add custom styles for use within the Web docs. For an example, see the [the SideNav](https://helios.hashicorp.design/components/side-nav?tab=code#layout) and custom styles for it in this file: "website/app/styles/pages/components/side-nav.scss"

Reply:
That's what I thought might be a better solution, thanks for the feedback! Fixed in https://github.com/hashicorp/design-system/pull/1876/commits/625a30d4f57ba08d675740c412433b641560fb27